assets/js/lib/model/hosts.test.js

```diff
@@ -0,0 +1,48 @@
+import { hostFactory } from '@lib/test-utils/factories';
+import { canDisableUnit, canEnableUnit } from './hosts';
+
+describe('hosts', () => {
+  describe('enabling units', () => {
+    it('should be able to enable a unit', () => {
+      const host = hostFactory.build({
+        systemd_units: [{ name: 'foo.service', unit_file_state: 'disabled' }],
+      });
+      expect(canEnableUnit(host, 'foo.service')).toBe(true);
+    });
+
+    it('should not be able to enable a unit', () => {
+      const host = hostFactory.build({
+        systemd_units: [
+          { name: 'foo.service', unit_file_state: 'enabled' },
+          { name: 'bar.service', unit_file_state: 'unknown' },
+          { name: 'baz.service', unit_file_state: 'unrecognized' },
```

Comment:
nitpick:
maybe more than `unrecognized`, `{ name: 'baz.service' }` and a test using `unknown.service` would cover more untested scenarios

Reply:
Covered more cases here f7f2228a9375b5456b068f704e4df43498073018